JamGame/Assets/Scripts/Employee/View.cs

```diff
@@ -27,10 +26,11 @@ public void RevertOverlays()
         }
     }
 
-    [RequireComponent(typeof(MeshRenderer))]
     public partial class View : IOverlayRenderer<Overlay.Stress>
     {
+        [SerializeField]
         private MeshRenderer meshRenderer;
```

Comment:
Maybe better change to List of mesh renderers for multiple parts of Employee?

Reply:
I think it can be implemented later, if needed